### PR TITLE
US-InsertVideo-02 : Video links preview after posting

### PR DESCRIPTION
**Pull request by Rdidmohamedelhadi**

### CODE REVIEW

File: commons-registration/pom.xml [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+    <groupId>org.exoplatform.commons</groupId>
+    <version>6.0.x-activity-composer-vue-SNAPSHOT</version>
+  </parent>
+  <artifactId>commons-registration</artifactId>
+  <packaging>war</packaging>
+  <name>eXo PLF:: Commons - Register Software on Tribe</name>
+  <build>
+    <finalName>registration</finalName>
+  </build>
+</project>
```
Comment (thomasdelhomenie) [inline commons-registration/pom.xml]: This file should be removed no ?

File: commons-welcome-screens-webapp/pom.xml [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+      <groupId>org.exoplatform.commons</groupId>
+      <version>6.0.x-activity-composer-vue-SNAPSHOT</version>
+    </parent>
+    <artifactId>commons-welcome-screens-webapp</artifactId>
+    <packaging>war</packaging>
+    <name>eXo PLF:: Commons - Welcome screens Webapp</name>
+    <build>
+        <finalName>welcome-screens</finalName>
+    </build>
+</project>
```
Comment (thomasdelhomenie) [inline commons-welcome-screens-webapp/pom.xml]: This file should be removed no ?

**Review by thomasdelhomenie — CHANGES_REQUESTED** [review]

File: commons-registration/pom.xml (hunk 1 above)
Comment (rdenarie) [reply commons-registration/pom.xml]: done

File: commons-welcome-screens-webapp/pom.xml (hunk 2 above)
Comment (rdenarie) [reply commons-welcome-screens-webapp/pom.xml]: done

**Review by rdenarie — DISMISSED** [review]
LGTM

**Review by rdenarie — APPROVED** [review]
LGTM

**Review by thomasdelhomenie — APPROVED** [review]

**Pull request merged**